File: C#/BluffinMuffin.Protocol/Lobby/JoinTableResponse.cs
using System.Collections.Generic;
using BluffinMuffin.Protocol.DataTypes;
using BluffinMuffin.Protocol.DataTypes.Attributes;

namespace BluffinMuffin.Protocol.Lobby
{
    /// <summary>
    /// 
    /// </summary>
    public class JoinTableResponse : AbstractBluffinReponse<JoinTableCommand>
    {
        /// <summary>
        /// Parameters of the poker table
        /// </summary>
        public TableParams Params { get; set; }

        /// <summary>
        /// The total amount of money currently played in this game. This include the money in the center, and the money played in front of each player.
        /// </summary>
        [ExampleValue(42000)]
        public int TotalPotAmount { get; set; }

        /// <summary>
        /// All the pots on the table. Usually only one, but can have more if some players are all-in.
        /// </summary>
        [ExampleValues(2, 4200, 420)]
        public List<int> PotsAmount { get; set; }

        /// <summary>
        /// All the card ids currently visible on the board (Preflop: empty, flop: 3 cars, turn: 4 cards, river: 5 cards) 
        /// </summary>
        [ExampleValues(3, "2s", "Kh", "5d")]
        public string[] BoardCards { get; set; }

        /// <summary>
        /// The information about every seats around the table
        /// </summary>
        [ExampleValues(1)]
        public List<SeatInfo> Seats { get; set; }

        /// <summary>
        /// Informs the client if the game is currently in the playing state, or not.
        /// </summary>
        [ExampleValue(true)]
        public bool GameHasStarted { get; set; }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public JoinTableResponse(JoinTableCommand command)
            : base(command)
        {
        }
    }
}

File: Python/bluffinmuffin/protocol/lobby/join_table_response.py
from bluffinmuffin.protocol.enums import BluffinMessageIdEnum
from bluffinmuffin.protocol.interfaces import AbstractResponse
from .join_table_command import JoinTableCommand
from bluffinmuffin.protocol.data_types import SeatInfo
from bluffinmuffin.protocol.data_types import TableParams


class JoinTableResponse(AbstractResponse):
    def __init__(self, success, message_id, message, jsonCommand, params, total_pot_amount, pots_amount, board_cards, seats, game_has_started):
        super().__init__(success, message_id, message, JoinTableCommand.decode(jsonCommand))
        self.params = params
        self.total_pot_amount = total_pot_amount
        self.pots_amount = pots_amount
        self.board_cards = board_cards
        self.seats = seats
        self.game_has_started = game_has_started

    def __str__(self):
        return '{0} => ({1}) {2} [{3}] [{4}] [{5}] {6}'.format(
            super().__str__(),
            self.params,
            self.total_pot_amount,
            ', '.join([x.__str__() for x in self.pots_amount]),
            ', '.join(self.board_cards),
            ', '.join([x.__str__() for x in self.seats]),
            self.game_has_started
        )

    def _encode_specific(self, d):
        super()._encode_specific(d)
        d['Params'] = self.params.encode()
        d['TotalPotAmount'] = self.total_pot_amount
        d['PotsAmount'] = self.pots_amount
        d['BoardCards'] = self.board_cards
        d['Seats'] = [x.encode() for x in self.seats]
        d['GameHasStarted'] = self.game_has_started

    @classmethod
    def decode(cls, obj):
        return cls(
            obj['Success'],
            BluffinMessageIdEnum.parse(obj['MessageId']),
            obj['Message'],
            obj['Command'],
            TableParams.decode(obj['Params']),
            obj["TotalPotAmount"],
            obj["PotsAmount"],
            obj["BoardCards"],
            [SeatInfo.decode(x) for x in obj['Seats']],
            obj['GameHasStarted']
        )
